libs/providers/flagd-web/README.md

```diff
@@ -0,0 +1,55 @@
+# Client-side flagD Provider for OpenFeature
+
+![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)
+
+Flagd is a simple daemon for evaluating feature flags.
```

Comment:
Update the flagd description.

Reply:
OK - I haven't reviewed any of this yet - but I will.